cli/src/set_variables.rs

```diff
@@ -0,0 +1,55 @@
+use std::path::PathBuf;
+
+use crate::ExecuteZeebeCommand;
+use async_trait::async_trait;
+use clap::Args;
+use color_eyre::Result;
+
+use zeebe_client::{
+    api::{SetVariablesRequest, SetVariablesResponse},
+    ZeebeClient,
+};
+
+#[derive(Args)]
+
+pub(crate) struct SetVariablesArgs {
+    #[clap(short, long)]
+    element_instance_key: i64,
+    #[clap(short, long)]
+    local: bool,
+    #[clap(long, value_parser, group = "value")]
+    path: Option<PathBuf>,
+    #[clap(long, group = "value")]
+    json: Option<String>,
+}
+
+impl TryFrom<SetVariablesArgs> for SetVariablesRequest {
+    type Error = color_eyre::Report;
+
+    fn try_from(args: SetVariablesArgs) -> Result<SetVariablesRequest, Self::Error> {
+        let variables = if let Some(path) = &args.path {
+            std::fs::read_to_string(path)?
+        } else if let Some(json) = args.json {
+            json
+        } else {
+            unreachable!()
```

Comment:
:wrench: Panic with a nice error message

Reply:
I guess the idea behind using `unreachable` was that clap guarantees that either path or json is set. Bummer that we can't model that with types (I think?). So yeah, a proper panic seems appropiate.

Reply:
I guess the idea behind using `unreachable` was that clap guarantees that either path or json is set. Bummer that we can't model that with types (I think?). So yeah, a proper panic seems appropriate.

Reply:
Will do it eventually